### PR TITLE
[ubuntu-dev] install the c9.ide.janitorconfig plugin for all images

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

File: ubuntu-dev/ubuntu-dev.dockerfile [hunk 1]
```diff
@@ -181,13 +181,14 @@ RUN git clone https://github.com/kanaka/noVNC /home/user/.novnc/ \
...
+ && git clone https://github.com/JanitorTechnology/c9.ide.janitorconfig \
  && git clone https://github.com/nt1m/c9.ide.reviewcomments \
  && cd /home/user/.c9sdk \
  && ./scripts/install-sdk.sh \
  && git checkout -- node_modules \
  && npm install -g c9
-ADD client-workspace-janitor.js /home/user/.c9sdk/configs/
-RUN sudo chown user:user /home/user/.c9sdk/configs/client-workspace-janitor.js
+ADD workspace-janitor.js /home/user/.c9sdk/configs/ide/
+RUN sudo chown user:user /home/user/.c9sdk/configs/ide/workspace-janitor.js
```
Comment (nt1m) [inline ubuntu-dev/ubuntu-dev.dockerfile]: r+ without this change

Comment (jankeromnes) [inline ubuntu-dev/ubuntu-dev.dockerfile]: Thanks! Maybe it was not what was breaking though. I'll try with it again.

Comment (jankeromnes) [inline ubuntu-dev/ubuntu-dev.dockerfile]: It works now!

**Comment by jankeromnes** [comment]
Ok, this pull request seems to work fine now.

I suggest that we add `janitor.json` config files to all projects in a follow-up pull request (that addresses my nits in https://gist.github.com/nt1m/d6d98cc9d83ae072d41842bcf396ae05#gistcomment-2151704), and merge this one (it will simply cause all projects not to have any run config as long as there is no `janitor.json` config file).

**Pull request merged**